dnf-docker-test/launch-test

```diff
@@ -3,5 +3,10 @@ set -xeuo pipefail
 
 httpd -k start
 new_name=${1}-${2}
-mv /behave/${1}.feature /behave/${new_name}.feature
+EXIT_STATUS=0
+cp /tests/${1}.feature /behave/${new_name}.feature || EXIT_STATUS=1
```

Comment:
I would like to do two things:
1. Print human understable what went wrong("Feature file ${1}.feature not found") 
2. End return with exit code 1.